packages/app/src/runs/modals/CloudConnectModals.spec.tsx

```diff
@@ -18,7 +19,16 @@ describe('<CloudConnectModals />', () => {
 
         result.cloudViewer = {
           ...CloudUserStubs.me,
-          organizations: noOrg ? null : CloudOrganizationConnectionStubs,
+          organizations: hasOrg ? cloneDeep(CloudOrganizationConnectionStubs) : null,
```

Comment:
Why do we need to cloneDeep here?

Reply:
The GQL stub isn't reset between tests since it's just a simple object import, since I'm mutating the data on lines 26-31 I didn't want my changes leaking into other tests

Reply:
@lmiller1990 Going to merge this PR so it doesn't sit too long, but let me know if you have an issue with this pattern or know of a better way to address it - happy to revisit this